source/features/table-input.tsx

```diff
@@ -0,0 +1,76 @@
+import './table-input.css';
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import TableIcon from 'octicon/diff-added.svg';
+import * as pageDetect from 'github-url-detection';
+import * as textFieldEdit from 'text-field-edit';
+
+import features from '.';
+import smartBlockWrap from '../helpers/smart-block-wrap';
+
+type TableDimensions = [number, number];
```

Comment:
Use named tuples.

Reply:
IMO it a tuple is named and exported as its own type, it might as well just be an object. 

So either change it to an object or just use it inline where needed without creating a `type`